.stylelintrc.yml

```diff
@@ -83,6 +83,10 @@ rules:
   unit-no-unknown: true
   # Disallow absolute paths for imports in bounds of library files
   esl-less/import-root: never
+  # Disallow long notation for hex colors
+  color-hex-length: short
+  # Disallow uppercase for hex colors
+  color-hex-case: lower
```

Comment:
I think it would be easier to understand if the rules were sorted alphabetically